### PR TITLE
kelly/economic-calendar

### PR DESCRIPTION
**Pull request by kelly-binary**

### CODE REVIEW

File: src/templates/mt5/economic_calendar.html.tt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div class="primary-color-dark cfd-header">
+    <div class="fx-panel center-text container">
+        <h1 class="content-inverse-color">[% l('Economic Calendar') %]</h1>
+        <p class="content-inverse-color">[% l('Check out the latest economic events and indicators from all over the world. All data are collected and updated in real time.') %]</p>
+    </div>
+</div>
+
+<div class="container">
+    <p>[% l('You can monitor macroeconomic indicators straight from the MetaTrader 5 platform using release time labels displayed on <a href="[_1]" target="_blank">financial symbol charts</a>.', request.url_for('/download/metatrader5_help.pdf')) %]</p>
```
Comment (mohammad-deriv) [inline src/templates/mt5/economic_calendar.html.tt]: File name used here `metatrader5_help.pdf` doesn't match the actual file name `MetaTrader5_Help.pdf`

Comment (kelly-binary) [reply src/templates/mt5/economic_calendar.html.tt]: @binary-hamedanchi you're right! thanks for catching that :D

**Review by mohammad-deriv — APPROVED** [review]

File: src/templates/mt5/economic_calendar.html.tt [hunk 2]
```diff
@@ -6,7 +6,7 @@
 </div>
 
 <div class="container">
-    <p>[% l('You can monitor macroeconomic indicators straight from the MetaTrader 5 platform using release time labels displayed on <a href="[_1]" target="_blank">financial symbol charts</a>.', request.url_for('/download/MetaTrader5_Help.pdf')) %]</p>
+    <p>[% l('You can monitor macroeconomic indicators straight from the MetaTrader 5 platform using release time labels displayed on <a href="https://www.metatrader5.com/en/terminal/help/fundamental?utm_source=www.mql5.com&utm_campaign=en.main.page.mql5#chart" target="_blank">financial symbol charts</a>.') %]</p>
```
Comment (binary-hamedanchi) [inline src/templates/mt5/economic_calendar.html.tt]: The link can be simply: `https://www.metatrader5.com/en/terminal/help/fundamental#chart`

File: src/templates/mt5/economic_calendar.html.tt [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<div class="primary-color-dark economic-calendar-header">
+    <div class="fx-panel center-text container">
+        <h1 class="content-inverse-color">[% l('Economic Calendar') %]</h1>
+        <p class="content-inverse-color">[% l('Check out the latest economic events and indicators from all over the world. All data are collected and updated in real time.') %]</p>
+    </div>
+</div>
+
+<div class="container">
+    <p>[% l('You can monitor macroeconomic indicators straight from the MetaTrader 5 platform using release time labels displayed on <a href="https://www.metatrader5.com/en/terminal/help/fundamental?utm_source=www.mql5.com&utm_campaign=en.main.page.mql5#chart" target="_blank">financial symbol charts</a>.') %]</p>
```
Comment (mohammad-deriv) [inline src/templates/mt5/economic_calendar.html.tt]: The link can be simply: `https://www.metatrader5.com/en/terminal/help/fundamental#chart`

**Review by mohammad-deriv — APPROVED** [review]

**Pull request merged**